wordpress-mobile/WordPressAuthenticator-iOS
# Reduces the number of Pasteboard alerts displayed by the 2FA screen

Addresses most of https://github.com/wordpress-mobile/WordPress-iOS/issues/15438.
WordPress PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/15454

* Adds new UIPasteboard detection methods on iOS 14.
* Uses `changeCount` instead of the contents to detect pasteboard changes.

### Testing

* Enter username and password for an account with 2FA enabled.
* On the 2FA screen leave the app - You should _not_ see any pop ups about pasteboard usage when leaving the app.
* Copy a URL from Safari.
* Come back to the app - You should _not_ see any alert for pasteboard usage.
* Leave the app and copy a 2FA code.
* Return to the app - You _should_ see an alert for pasteboard usage and the code should be pasted into the field.
